### PR TITLE
Sign up form from mint.com

### PR DESCRIPTION
**Pull request by Amita-Roy**
I have created this sign up form in collaboration with my partner.

We have used icons from fontawesome

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                Standard call, messaging or data rates may apply.
+              </p>
+              <label class="password-label" for="password">Password</label>
+              <input type="password" name="password" id="password" /><span
+                class="password-icon"
+                ><i class="fas fa-lock-open"></i
+              ></span>
+              <button class="createAcc-btn">
+                <span><i class="fas fa-lock"></i></span> Create Account
+              </button>
```
Comment (HADDADSOHAIB) [inline index.html:64]: The button should have a type='submit':
- [ ] It has a `<submit>` button that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

File: index.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+              >Learn More</a
+            >
+          </div>
+
+          <div class="signUp-form">
+            <form>
+              <label class="email-label" for="email">Email address</label>
+              <input type="email" name="email" id="email" />
+              <label class="phone-label" for="phone">Phone (recommended)</label>
+              <input type="text" name="phone" id="phone" class="phone" />
```
Comment (HADDADSOHAIB) [inline index.html:53]: This input should have a type='tel':
- [ ] It has an `<input>` of type `tel` and a label `Phone` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

```

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Pull request merged**